From stable-bounces@linux.kernel.org Fri Mar 31 13:09:30 2006
Date: Fri, 31 Mar 2006 23:09:03 +0200
From: Adrian Bunk <bunk@stusta.de>
To: stable@kernel.org
Message-ID: <20060331210903.GC22677@stusta.de>
Content-Disposition: inline
Cc: maximilian attems <maks@sternwelten.at>
Subject: isicom must select FW_LOADER

From: maximilian attems <maks@sternwelten.at>

The isicom driver uses request_firmware()
and thus needs to select FW_LOADER.

This patch was already included in Linus' tree.

Signed-off-by: maximilian attems <maks@sternwelten.at>
Signed-off-by: Adrian Bunk <bunk@stusta.de>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/char/Kconfig |    1 +
 1 file changed, 1 insertion(+)

--- linux-2.6.16.1.orig/drivers/char/Kconfig
+++ linux-2.6.16.1/drivers/char/Kconfig
@@ -187,6 +187,7 @@ config MOXA_SMARTIO
 config ISI
 	tristate "Multi-Tech multiport card support (EXPERIMENTAL)"
 	depends on SERIAL_NONSTANDARD
+	select FW_LOADER
 	help
 	  This is a driver for the Multi-Tech cards which provide several
 	  serial ports.  The driver is experimental and can currently only be
